tests/simulator/test_diagnostics.py

```
@@ -20,9 +22,16 @@
 
 from tests.simulator import SimulatorTest
 from tests.diagnostic import dump_all_diags
+from tests.simulator import SimulatorTest
```

Ruff (0.15.2)

[error] Redefinition of unused `SimulatorTest` from line 23: `SimulatorTest` redefined here

Remove definition: `SimulatorTest`

(F811)